app/views/devise/registrations/new.html.erb

```diff
@@ -1,12 +1,19 @@
 <div class="flex flex-col gap-2 relative text-white">
   <h2 class="bg-blue-800 text-center py-4">
-  <i class="fa-solid fa-arrow-left float-left mt-1 ml-2"></i>
+  <a href="<% link_to 'home', root_path %>" >
+    <i class="fa-solid fa-arrow-left float-left mt-1 ml-2"></i>
+  </a>
   <span class="-ml-4">Log in</span>
   </h2>
   <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
     <%= render "devise/shared/error_messages", resource: resource %>
 
-    <div class="my-1">
+    <div class="p-2 w-full bg-white flex gap-4 items-center text-gray-400">
+    <i class="fa-solid fa-image-portrait text-5xl rounded-lg"></i>
+    <span>Upload image</span>
+    </div>
+
+    <div class="mb-1">
       <%= f.text_field :full_name, autofocus: true, autocomplete: "full_name", class: 'border-none focus:border-none bg-white', placeholder: 'Full Name' %>
```

Comment:
- [ ] It would be better if a user can see what he or she types in the input fields. It is very difficult entering the matching fields and logging into the account, I'm basically blind on the screen